docs/markdown/Reference-manual.md

```diff
@@ -822,6 +832,10 @@ overridden by specifying it with the `install_dir` keyword
 argument. All man pages are compressed during installation and
 installed with a `.gz` suffix.
 
+The `install_mode` argument can be used to specify the file mode in symbolic
+format and optionally the owner/uid and group/gid for the installed files.
+*(Added 0.47.0)*.
+
```

Comment:
These two should have an example of usage to clarify what "optionally owner/uid and group/gid" actually means. Something like "An example value could be `['rwxr-sr-x', 'root', 'root']`".

Reply:
Done.

One good idea would be to document common options (such as these) in a single common section.

If I have some free time I'll try to propose that refactor as a PR...

Cheers,
Filipe